### PR TITLE
OCPBUGS-18132: Add patch for allowing configmap updates via clusterrole

### PR DESCRIPTION
**Pull request by gnufied**
Goes with https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/169

Fixes OCPBUGS-18132

### CODE REVIEW

File: assets/csidriveroperators/vsphere/06_clusterrole.yaml [hunk 1]
```diff
@@ -309,6 +309,8 @@ rules:
   - get
   - list
   - watch
+  - patch
+  - update
```
Comment (jsafrane) [inline assets/csidriveroperators/vsphere/06_clusterrole.yaml:313]: This allows the vSphere CSI driver operator to patch any pod and ConfigMap in any namespace. IMO it's too broad.
It should allow only selected ConfigMap in openshift-config-managed namespace, i.e. a Role could be better. And definitely decoupled from Pods.

Comment (gnufied) [reply assets/csidriveroperators/vsphere/06_clusterrole.yaml]: Moved this to use named configmap. I think that works as well.

**Comment by duanwei33** [comment]
See test on https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/169.

**Pull request merged**